main.c

```diff
@@ -408,23 +381,20 @@ from_int(PyObject *a)
     return res;
 }
 
-
 static int
 to_bool(MPZ_Object *a)
 {
     return a->size != 0;
 }
 
-
-#define SWAP(T, a, b)  \
-    do {               \
-        T tmp = a;     \
-        a = b;         \
-        b = tmp;       \
+#define SWAP(T, a, b) \
+    do {              \
+        T tmp = a;    \
+        a = b;        \
+        b = tmp;      \
```

Comment:
Why to change this?  IIUC, the "\" char appears after just one space after longest line.  That doesn't make much sense for me.